cumulusci/tasks/marketing_cloud/deploy.py

```diff
@@ -171,10 +189,11 @@ def _validate_response(self, deploy_info: dict):
 
         self.logger.info("Deployment completed successfully.")
 
-    def _report_fatal_error(self, result: dict):
+    def _report_error(self, response_data: dict):
+        deploy_status = response_data["status"]
         self.logger.error(
-            f"> {DEPLOY_ERROR_STATUS} received. Dumping response from Marketing Cloud:\n{result}"
+            f"Received status of: {deploy_status}. Received the following data from Marketing Cloud:\n{response_data}^"
```

Comment:
Just validating: there's no indication that this response could contain confidential data, correct?

Reply:
I believe I looked at that with Marcel last spring when we implemented this task, but I don't recall what the actual response body for an errored deployment looks like.